### PR TITLE
Update allow scripts

### PR DESCRIPTION
**Pull request by leotm**
- Support Yarn 3+ (Berry) via plugin
- Add `list` to CLI help menu
- Update console log for `--experimental-bins`

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -48,6 +48,6 @@
     }
   ],
   "env": {
-    "es6": true
+    "es2020": true
```
Comment (legobeat) [inline .eslintrc.json:51]: What's the motivation here? If reasonable it seems preferrable to keep backwards-compatatibility for this package.

File: packages/allow-scripts/src/yarn-berry-plugin.cjs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// @ts-check
+
+const { execute } = require( "@yarnpkg/shell");
```
Comment (legobeat) [inline packages/allow-scripts/src/yarn-berry-plugin.cjs]:
```suggestion
const { execute } = require('@yarnpkg/shell');
```

**Comment by legobeat** [comment]
@leotm I know this is still draft but can we split the yarn berry changes in a separate PR from the CLI changes?

File: .eslintrc.json (hunk 1 above)
Comment (boneskull) [reply .eslintrc.json:51]: template literals, from looking at the changes.  lavamoat doesn't support any version of node that doesn't have template strings anyway.  `es2020` is equivalent to node.js v14.17.0+ and node 14.x is EOL anyway

**Comment by legobeat** [comment]
@leotm just fyi i rebased this over at #766

**Comment by leotm** [comment]
> @leotm just fyi i rebased this over at #766

thanks 🙈 closing here then regathering thoughts to finish over in 766

**Pull request closed without merging**